doc/classes/BoxShape.xml

```diff
@@ -4,7 +4,7 @@
 		Box shape resource.
 	</brief_description>
 	<description>
-		Box shape resource, which can be set into a [PhysicsBody] or area.
+		Can be a child of a [PhysicsBody], [Area], or [Area2D].
```

Comment:
This is a 3D shape, I don't think it can be a child of Area2D.

Reply:
Just checked through the source code again and you're right, the 3D collision shape includes this class but the 2D one doesn't. Will correct.